### DISCUSSION TITLE
using $HOME in the config file

### DISCUSSION THREAD

**Discussion by mil-ad** [ahrm/sioyek#690] [Q&A]
I've now put my shared db in something like dropbox and want to set `shared_database_path` in the config. Ideally I'd like to use the same config files across all my machines but on Linux my dropbox is `/home/my_name/dropbox` but on macOS it's `/Users/my_name/dropbox`. Do we something that expands to home directory in the config? `$HOME` doesn't seem to work.

**Comment by ahrm** [comment]
Doesn't `~/my_name/dropbox` work?

**Comment by mil-ad** [comment]
crashes on macOS

**Comment by mil-ad** [comment]
I'm on 2.0.0 btw
